new-era-commands/rotations/email.js

```diff
@@ -0,0 +1,3 @@
+const { rotationBuilder } = require("../../utils/rotation-builder");
+
+module.exports = rotationBuilder("email", "emailRotationList");
```

Comment:
Hmm I would prefer the names be matched to the _reason_ for the rotation rather than just the group that it might apply to. Specifically, because I don't think we'd want to limit the number of rotations to just core/maint/mod, and I don't think it's unlikely that in the future there's a rotation that crosses those boundaries.

I'm not dead set on email and triage, but would be opposed to maintainerrotation etc.